code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -518,3 +518,38 @@
 	desc = "For when you're stepping on up to the plate."
 	icon_state = "JackFrostShoes"
 	inhand_icon_state = "JackFrostShoes_worn"
+
+/obj/item/clothing/shoes/gunboots //admin boots that fire gunshots randomly while walking
+	name = "gunboots"
+	desc = "This is what all those research points added up to, the ultimate workplace hazard."
+	icon_state = "jackboots"
+	inhand_icon_state = "jackboots"
```

Comment:
if they're gonna look like jackboots and even have the same pockets as jackboots, why not just make these subtypes of jackboots?